packages/api/src/schema/schema.yaml

```diff
@@ -2572,6 +2572,70 @@ components:
           type: number
           example: 200000
 
+    create-room-response:
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+
+    room:
```

Comment:
Would you mind adding documentation (`description`) to the user-facing fields in this object?